docs/integration-guides/wordpress.md

```diff
@@ -8,28 +8,32 @@ keywords:
 pagination_next: null
 pagination_prev: null
 last_update: 
-   date: 07/07/2023
-   author: Patricia McPhee
+   date: 08/07/2023
+   author: Anna Garcia
 draft: false
 doc_type: how-to
 displayed_sidebar: mainSidebar
 ---
 
+import ProtocolOIDC from '../includes/_protocol_oidc.mdx';
+import HostedWeb from '../includes/_hosted-web.mdx';
+import ClientTypeConfidential from '../includes/_client-type_confidential.mdx';
+import GrantTypeAuthorizationCode from '../includes/_grant-type_authorization-code.mdx';
 
 
-This guide provides information on how to set up Beyond Identity as a passwordless authentication provider for a Drupal admin panel login.
+This guide provides information on how to set up Beyond Identity as a passwordless authentication provider for a Wordpress site.
  
 
 In this guide, you'll:
 
-- Configure Beyond Identity as an Identity Provider
-- Install and configure the required plugin to support an OpenID Connect (OIDC) integration
+1. Configure Beyond Identity as an Identity Provider
+1. Install and configure the required plugin to support an OpenID Connect (OIDC) integration
```

Comment:
nit: Was this an intended change? I notice in the other integration guides, we have a bulleted list instead of a numbered list

Reply:
This was intentional. I thought numbers were nice to see how many steps are required

Reply:
In that case, should we update the others as well, for consistency?

Reply:
Hm that might be a good idea

Reply:
Found a couple of places that do this and made them consistent.